templates/garden/nightly/classifier-resnet.libsonnet

```diff
@@ -73,11 +73,19 @@ local tpus = import "../../tpus.libsonnet";
   local v3_8 = {
     accelerator: tpus.v3_8,
   },
+  local v2_32 = {
+    accelerator: tpus.v2_32,
+  },
+  local v3_32 = {
+    accelerator: tpus.v3_32,
+  },
 
   configs: [
     resnet + v2_8 + functional,
     resnet + v3_8 + functional,
     resnet + v2_8 + convergence,
     resnet + v3_8 + convergence,
+    resnet + v2_32 + convergence,
```

Comment:
Please make these experimental for now. Automated pod tests are coming soon ™️

Reply:
Resolving this conversation as it should be obselete.